server/utils/index.js

```diff
@@ -14,6 +14,30 @@ const getEstablishmentId = (
     10,
   );
 
+const getEstablishment = (
+  agencyId,
+  establishmentData = defaultEstablishmentData,
+) => {
+  const establishmentId = Object.keys(establishmentData).find(
+    id => establishmentData?.[id]?.agencyId === agencyId,
+  );
+  return {
+    establishmentId,
+    establishmentName: establishmentData[establishmentId]?.name,
+  };
+};
+
+const updateSessionEstablishment = (req, agencyId) => {
```

Comment:
would be nice to have tests for these functions independently of the integrations tests